libs/application/templates/children-residence-change/src/fields/ChangeInformation/index.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react'
 import { useIntl } from 'react-intl'
 import { Box, Text } from '@island.is/island-ui/core'
 import { CheckboxController } from '@island.is/shared/form-fields'
-import { constructParentAddressString } from '../../lib/utils'
+import { constructAddressString } from '../../lib/utils'
```

Comment:
Wondering if we should rename this function. What do you think? 
I am thinking about something like `formatHomeAddress`

Reply:
Yeah, I just removed the `parent` bit because that didn't make sense but I agree with `formatHomeAddress` or even just `formatAddress` since it can be whatever address really.